spec/requests/flows/revoke_token_spec.rb

```diff
@@ -127,14 +131,18 @@
         it "should not revoke the access token provided" do
           post revocation_token_endpoint_url, params: { token: access_token.token }
 
-          expect(response).to be_successful
+          expect(response).to be_forbidden
+          expect(response.body).to include("unauthorized_client")
+          expect(response.body).to include("You are not authorized to revoke this token")
```

Comment:
Here we can also check using i18n. Doorkeeper is a multilanguage solution so we must support internationalization :)